make_release.rb

```diff
@@ -203,6 +206,12 @@ def prepare_readme(target_folder)
   FileUtils.cp REVISION_FILE, target_folder
 end
 
+# Copies desktop file & icon to the target folder
+def prepare_desktop(target_folder)
+  FileUtils.cp DESKTOP_FILE, target_folder
+  FileUtils.cp ICON_FILE, target_folder + "/Thrive.png"
```

Comment:
`File.join` should be used instead of assuming target folder doesn't end with `/`

Reply:
Ah, you're right, I didn't think to use `join()`; I'll fix this in a minute and update the PR :+1:

That said, two `/` in a row make no difference on any of the Unixes I've seen :)